serverscommands/flavorcommands/get.go

```diff
@@ -40,5 +41,12 @@ func commandGet(c *cli.Context) {
 
 func tableGet(c *cli.Context, i interface{}) {
 	keys := []string{"ID", "Name", "Disk", "RAM", "RxTxFactor", "Swap", "VCPUs"}
-	output.MetaDataTable(c, i, keys)
+	f := func() map[string]interface{} {
+		return singleFlavor(i)
+	}
```

Comment:
Actually, since all take a function pointer with this, why not pass `nil` or... heck just `structs.Map`?